group-generators/generators/wiw-crypto-memento/index.ts

```diff
@@ -0,0 +1,44 @@
+// sismo-hub/group-generators/generators/wiw-crypto-memento/index.ts
```

Comment:
remove the commentary from the tutorial (same remark for the other group generator)

Reply:
Done.